### ISSUE TITLE
Disconnecting from VPN after starting a VM leads to an error preventing next run

### ISSUE THREAD

**Issue by anjannath** [cirruslabs/tart#475]
if i started a VM when i was connected to a VPN and after that shutdown the VM and disconnect from VPN and then try to start the same VM again, it ends up with the following error:

```
❯ tart run vm-test
virtual machine's network attachment <VZNetworkDevice: 0x6000002ebbe0> has been disconnected with error: Error Domain=VZErrorDomain Code=1 "Internal Network Error." UserInfo={NSLocalizedFailure=Internal Virtualization error., NSLocalizedFailureReason=Internal Network Error.}
```

maybe it tries to add the tap device to the VM that was created by the VPN connection, but it doesn't exist when disconnected from VPN and `tart` ends up with the above error?

**Comment by fkorotkov** [comment]
Hey @anjannath, could you please share how you create this `vm-test`? Is it Linux or macOS VM? Did you use Tart Packer plugin? What's you host macOS version is?

**Comment by fkorotkov** [comment]
Closing for now since there is not enough information to assist.

**Comment by dronenb** [comment]
Hi @fkorotkov, I also saw this issue when I was connected to a WireGuard VPN and ran the commands on the README:
```
brew install cirruslabs/cli/tart
tart clone ghcr.io/cirruslabs/macos-ventura-base:latest ventura-base
tart run ventura-base
```
First time ever trying `tart`. Disconnecting from VPN did not help, only once I rebooted did the issue go away. I then tried deleting all local VM's, connected to WireGuard VPN,  and re-cloining the ventura-base image. Could not reproduce.
